plugins/grafana-custom-plugins/grafana-http-table-plugin/cypress/integration/01-smoke.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { e2e } from '@grafana/e2e';
+
+e2e.scenario({
```

Comment:
Do we need this e2e test for Grafana? 
cc @heanlan

Reply:
Can we run it with automation? e.g. will github action trigger it?

Reply:
I can remove this, I don't think it's testing anything important, nor is it run during yarn test.